.github/.github/pull_request_template.md

```diff
@@ -0,0 +1,54 @@
+# 📄 PR Description
+
+[Add a description for this pull request. Follow the checklist below.] 
+<!--- Describe your changes in detail including a relevant motivation and context for this change -->
+
+
+# ✒️ Checkliste PR-Author
+
+## Before submitting the PR, please make sure you do the following
+
+### Prepare Code for PR
+* [ ] The automated tests were successfully executed locally (`mvn clean test`).
+* [ ] All commented-out code and debug statements have been removed.
+* [ ] A test is included that fails without this PR but passes with it.
+* [ ] The relevant documentation, including code comments and public documentation, has been updated.
```

Comment:
```suggestion
* [ ] The documentation in the code has been updated and corresponds to the team guidelines
* [ ] A person has been assigned to update the public documentation as soon as possible if this PR requires changes to it
```

What do you think about this instead of the suggestion above?